### ISSUE TITLE
Ability to log-in via platforms other than Google

### ISSUE THREAD

**Issue by ethulin** [mefee/mefee#21]
I think people will appreciate being able to either set up their own account or use accounts other than Google. Probably a decent-sized feature request, so just putting it on the list.

I wonder if there is some somewhat off the shelf handling of these login regimes.

**Comment by harry-peirse** [comment]
Yes, this should be relatively straightforward as we've gone with firebase, which is just about as close to off-the-shelf as you can get for offloading authentication.
